tests/sync/test_llh.py

```diff
@@ -0,0 +1,14 @@
+from utils import get_api_key
```

Comment:
I'd suggest not using llh for file name. it's usually easier to have the same naming between code/tests